domains/_vercel.runkang10.json

```diff
@@ -0,0 +1,11 @@
+{
+  "owner": {
+    "username": "Runkang10",
+    "email": "chen.runkang10@gmail.com",
+    "discord": "runkang10"
+  },
+  "records": {
+    "TXT": "vc-domain-verify=runkang10.is-a.dev,f3afb8b4605d5f333957"
+  },
+  "proxied": true
```

Comment:
This file only has TXT records therefore it cannot be proxied. This should be removed or set to false.